### PR TITLE
Restart services on reboot

### LINKED ISSUE #5: Escualo should restart services after machine restart

### LINKED ISSUE #6: Escualo should restart services if they die unexpectedly

### PR DESCRIPTION
**Pull request by faloi**
Closes #5 
Closes #6 

I solved it using [Upstart](http://upstart.ubuntu.com/), who handles process respawn and start upon reboot.

### CODE REVIEW

File: bootstrap [hunk 1]
```diff
@@ -30,7 +30,10 @@ done
 echo "Add Mongo support?"
 select answer in "Yes" "No"; do
   case $answer in
-    Yes ) MONGO_SUPPORT="apt-get install -y mongodb"; break;;
+    Yes ) MONGO_SUPPORT=" \
+      apt-get install -y mongodb; && \
+      echo '' >> /etc/init/mongodb; && \
```
Comment (flbulgarelli) [inline bootstrap]: why '' >> ?

File: service [hunk 2]
```diff
@@ -25,43 +25,44 @@ cat > /var/scripts/$SERVICE/init <<EOF
...
+  start on runlevel [2345]
+  stop on runlevel [!2345]
 
-chmod +x /var/scripts/$SERVICE/{init, stop, start}
+  respawn
+
+  chdir /var/www/$SERVICE
+
+  script
+    for var in /root/.escualo/vars/*; do . \\\$var; done
```
Comment (flbulgarelli) [inline service]: Why do you embedd this script instead of calling .escualorc?

Comment (flbulgarelli) [inline service]: I mean, the line 

```
   for var in /root/.escualo/vars/*; do . \\\$var; done
```

Is already there

File: bootstrap (hunk 1 above)
Comment (faloi) [inline bootstrap]: Because the file is created by the installer, but it doesn't have the `respawn` option set.

File: service (hunk 2 above)
Comment (faloi) [inline service]: I know, I copied it from there. It seems the `source` doesn't work when called from Upstart, so I replaced it with the dot (don't really know what it means, just copied from some Stack Overflow answer).

**Pull request merged**